libsplinter/Cargo.toml

```diff
@@ -48,7 +48,7 @@ mio-extras = "2"
 openssl = "0.10"
 percent-encoding = { version = "2.0", optional = true }
 protobuf = "2"
-rand = { version = "0.7", optional = true }
+rand = "0.7"
```

Comment:
This should probably still be optional and required by the "circuit-template" feature.

Reply:
The `generate_random_base62_string` function that uses `rand` isn't used by the `circuit-template` feature; it's used by the circuit and service builders in the admin service, which aren't behind any feature flags.